app/models/loader/types.rb

```diff
@@ -96,43 +96,58 @@ def create_resource!
           resource.annotations_dataset.import(%i(resource_id name value), records)
         end
       end
-      
+
       def resource
         ::Resource[resourceid]
       end
     end
-      
+
     class Record < Types::Base
       include CreateRole
       include CreateResource
-      
+
+      def verify
+        message = "Verify method for entity #{self} does not exist"
```

Comment:
Isn't this going to cause failures on create? Looks like `create` is currently calling it.

Reply:
No - because it is executing the verify method of derived classes, and it is implemented  in all of them now